sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/java/com/azure/spring/sample/aad/controller/RoleController.java

```diff
@@ -23,4 +23,11 @@ public String group1() {
     public String group2() {
         return "group2 message";
     }
+
+    @GetMapping("Admin")
+    @ResponseBody
+    @PreAuthorize("hasRole('ROLE_Admin')")
```

Comment:
According to email `FastTrack for Azure - Feedback on v3 AAD Azure Spring starter`, it should be `APPROLE_Admin`